packages/babel-helpers/src/helpers.js

```diff
@@ -1102,7 +1102,21 @@ helpers.classStaticPrivateFieldSpecSet = helper("7.0.2")`
     descriptor.value = value;
     return value;
   }
-  
+`;
+
+helpers.classStaticPrivateMethodGet = helper("7.3.2")`
+  export default function _classStaticPrivateMethodGet(receiver, classConstructor, descriptor) {
+    if (receiver !== classConstructor) {
+      throw new TypeError("Private static access of wrong provenance");
+    }
+    return descriptor.call(classConstructor);
```

Comment:
This should just return the method (without calling it), and then we should do `.call(receiver, arg1, arg2)`. Otherwise these cases don't work:

```js
var method = foo.#method;
var result = foo.#method(1, 2, 3);
```